kubernetes/client/api/custom_objects_api.py

```diff
@@ -1918,11 +1918,13 @@ def list_cluster_custom_object(self, group, version, plural, **kwargs):  # noqa:
         :param str version: The custom resource's version (required)
         :param str plural: The custom resource's plural name. For TPRs this would be lowercase plural kind. (required)
         :param str pretty: If 'true', then the output is pretty printed.
+        :param bool allow_watch_bookmarks: allowWatchBookmarks requests watch events with type \"BOOKMARK\". Servers that do not implement bookmarks may ignore this flag and bookmarks are sent at the server's discretion. Clients should not assume bookmarks are returned at any specific interval, nor may they assume the server will send any BOOKMARK event during a session. If this is not a watch, this field is ignored. If the feature gate WatchBookmarks is not enabled in apiserver, this field is ignored.
```

Comment:
please add a API change release note to CHANGELOG.md: 
```
- Add allowWatchBookmarks, resoureVersionMatch parameters to custom objects. [kubernetes-client/gen#180](https://github.com/kubernetes-client/gen/pull/180)
```

Reply:
Fixed.